Octokit.Tests/Clients/RepositoryInvitationsClientTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using NSubstitute;
+using Octokit;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+public class RepositoryInvitationsClientTests
+{
+    public class TheGetAllForRepositoryMethod
+    {
+        [Fact]
+        public async Task RequestsCorrectUrl()
+        {
+            var connection = Substitute.For<IApiConnection>();
+            var client = new RepositoryInvitationsClient(connection);
+
+            await client.GetAllForRepository(1);
+
+            connection.Received().GetAll<RepositoryInvitation>(Arg.Is<Uri>(u => u.ToString() == "repositories/1/invitations"), "application/vnd.github.swamp-thing-preview+json");
+        }
+    }
+
+    public class TheGetAllForCurrentMethod
+    {
+        [Fact]
+        public async Task RequestsCorrectUrl()
+        {
+            var connection = Substitute.For<IApiConnection>();
+            var client = new RepositoryInvitationsClient(connection);
+
+            await client.GetAllForCurrent();
+
+            connection.Received().GetAll<RepositoryInvitation>(Arg.Is<Uri>(u => u.ToString() == "user/repository_invitations"), "application/vnd.github.swamp-thing-preview+json");
+        }
+    }
+
+    public class TheAcceptMethod
+    {
+        [Fact]
+        public async Task RequestsCorrectUrl()
+        {
+            var connection = Substitute.For<IApiConnection>();
+            var client = new RepositoryInvitationsClient(connection);
+
+            await client.Accept(1);
+
+            connection.Received().Patch(Arg.Is<Uri>(u => u.ToString() == "user/repository_invitations/1"), "application/vnd.github.swamp-thing-preview+json");
+        }
+    }
+
+    public class TheDeclineMethod
+    {
+        [Fact]
+        public async Task RequestsCorrectUrl()
+        {
+            var connection = Substitute.For<IApiConnection>();
+            var client = new RepositoryInvitationsClient(connection);
+
+            await client.Decline(1);
+
+            connection.Received().Delete(Arg.Is<Uri>(u => u.ToString() == "user/repository_invitations/1"), Arg.Any<object>(), "application/vnd.github.swamp-thing-preview+json");
+        }
+    }
+
+    public class TheDeleteMethod
+    {
+        [Fact]
+        public async Task RequestsCorrectUrl()
+        {
+            var connection = Substitute.For<IApiConnection>();
+            var client = new RepositoryInvitationsClient(connection);
+
+            await client.Delete(1, 2);
+
+            connection.Received().Delete(Arg.Is<Uri>(u => u.ToString() == "repositories/1/invitations/2"), Arg.Any<object>(), "application/vnd.github.swamp-thing-preview+json");
+        }
+    }
+
+    public class TheEditMethod
```

Comment:
should probably have a unit test for when `InvitiationUpdate` parameter is `null`